pkgs/development/tools/hostess/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "hostess";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "cbednarski";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1izszf60nsa6pyxx3kd8qdrz3h47ylm17r9hzh9wk37f61pmm42j";
+  };
+
+  subPackages = ["."];
+
+  vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
+  meta = with lib; {
+    description = "An idempotent command-line utility for managing your /etc/hosts* file.";
+    license = licenses.mit;
+    maintainers = with maintainers; [ "edlimerkaj" ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ edlimerkaj ];
```

Reply:
Thank you so much for your review. I have added myself to the maintainers list and made the suggested change.

Reply:
You should push your changes to branch `hostess-0.5.2` (and not other branches), otherwise the changes won't be applied to this pull request. Also please follow the contributing guide when naming your commits, you can drop some commits / squash some commits / update the commit message for some commits by using `git rebase` (https://docs.github.com/en/get-started/using-git/about-git-rebase), then push your changes with `--force` option.

Reply:
done